e2e/tests/test-1.spec.ts

```
@@ -0,0 +1,5 @@
+import { test, expect } from '@playwright/test';
+
+test('test', async ({ page }) => {
```

ESLint

[error] Unexpected global mocha test.

(ghost/mocha/no-global-tests)